### PR TITLE
MAT-5097 logging improvement

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## PR Template

Jira Ticket: [MAT-5097](https://jira.cms.gov/browse/MAT-5097)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included
* [x] No extemporaneous files are included (i.e. Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/api/MeasureServiceApi.ts [hunk 1]
```diff
@@ -19,7 +19,10 @@ export class MeasureServiceApi {
       console.log("Measure transferred successfully.");
       return response.data;
     } catch (error: any) {
-      console.log("Failed to transfer the measure over to MADiE: ", JSON.stringify(error.response?.data));
+      console.log(
+        "Failed to transfer the measure (measure namee: " + measure.measureName + ") over to MADiE: ",
```
Comment (adongare) [inline lambda-function/src/api/MeasureServiceApi.ts]: looks like typo in word: name**e**

**Review by adongare — DISMISSED** [review]

File: lambda-function/src/api/MeasureServiceApi.ts (hunk 1 above)
Comment (sb-cecilialiu) [reply lambda-function/src/api/MeasureServiceApi.ts]: Yeah you are right, sorry about it. Just corrected it. Thanks!

**Review by adongare — APPROVED** [review]
